lib/cli/kit/command_registry.rb

```diff
@@ -0,0 +1,79 @@
+require 'cli/kit'
+
+module CLI
+  module Kit
+    module CommandRegistry
```

Comment:
Some modest gymnastics here to manage variables on this command vs. access the module it's [extended in](https://github.com/Shopify/cli-kit-example/blob/master/lib/todo.rb#L12).